app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+
+const { getTopics ,
+      } = require("./controllers/topicsController");
+
+const app = express();
+app.use(express.json());
```

Comment:
This doesnt need to be added yet as your not parsing any of the response bodies just yet :)